execution/contextTree.sh

```diff
@@ -1324,9 +1327,9 @@ function upgrade_cmdb_repo_to_v1_3_1() {
   local -A account_mappings
   readarray -t account_files < <(find "${GENERATION_DATA_DIR}" -type f -name "account.json" )
   for account_file in "${account_files[@]}"; do
```

Comment:
This needs the same default ing logic as at line 1223/4.

Reply:
@ml019 update the required with commit 25f2908